projects/step-frontend/src/lib/plugins-initializer/register-microfrontend-plugins.ts

```diff
@@ -48,50 +42,21 @@ const loadModule = async (definition: MicrofrontendPluginDefinition): Promise<Pl
   return result;
 };
 
-const compileModule = async ({ compiler, injector, declaration, definition }: CompileCtx & PluginCtx): Promise<any> => {
-  try {
-    const moduleFactory = await compiler.compileModuleAsync(declaration.PluginModule);
-    const moduleRef = moduleFactory.create(injector);
-    return {
-      moduleClass: declaration.PluginModule,
-      moduleInstance: moduleRef.instance,
-    };
-  } catch (e) {
-    console.error(`Angular 2+ module "${definition.entryPoint}" compilation fail`, e);
-    return undefined;
-  }
-};
-
 export const registerMicrofrontendPlugins = async (
   pluginsDefinitions: MicrofrontendPluginDefinition[],
   compileCtx: CompileCtx
-): Promise<unknown> => {
+): Promise<void> => {
   if (pluginsDefinitions.length === 0) {
-    return undefined;
+    return;
   }
 
   const loadModules = pluginsDefinitions.map((def) => loadModule(def));
   const pluginCtxs = (await Promise.all(loadModules)).filter((x) => !!x) as PluginCtx[];
 
-  const compileModules = pluginCtxs.map((ctx) => compileModule({ ...compileCtx, ...ctx }));
-  const modules = (await Promise.all(compileModules)).filter((x) => !!x);
-
-  const modulesWithInit = modules
-    .map((m) => m.moduleInstance as Partial<PluginOnInit>)
-    .filter((m) => !!m.pluginOnInit) as PluginOnInit[];
-
-  if (modulesWithInit.length > 0) {
-    await Promise.all(modulesWithInit.map((m) => m.pluginOnInit()));
-  }
-
-  const hybridModules = modules
-    .map((module) => getPluginMetaInfo(module.moduleClass)?.hybridModuleName)
-    .filter((x) => !!x);
-
-  if (hybridModules.length > 0) {
-    const hostModule = getAngularJSGlobal().module(AJS_MODULE);
-    hostModule.requires.push(...hybridModules);
-  }
+  const compileModules = pluginCtxs.map((ctx) =>
+    compileModule(ctx.definition.entryPoint, ctx.declaration.PluginModule, compileCtx)
+  );
+  const modules = (await Promise.all(compileModules)).filter((x) => !!x) as CompiledModule[];
```

Comment:
https://trello.com/c/p2rvxtAO/171-vs-boolean

Reply:
I'd say for now either leave it as is or perhaps if you like Boolean() more you could use it instead, however I think this is something worth discussing and following everywhere